Possible missing image for version 3.1.2

_Please use this to only for bug reports. For questions or when you need help, you can use the [GitHub Discussions](https://github.com/strimzi/strimzi-kafka-operator/discussions), our [#strimzi Slack channel](https://slack.cncf.io/) or out [user mailing list](https://lists.cncf.io/g/cncf-strimzi-users/topics)._

**Describe the bug**
I have tried installing the new helm chart and values file. Which references the 0.31.1 docker image. When trying to install the operator it fails with 

**To Reproduce**
Steps to reproduce the behavior:
1. Download the strimzi repo `helm repo add strimzi https://strimzi.io/charts/ `
2. Use the values file found here https://github.com/strimzi/strimzi-kafka-operator/blob/main/helm-charts/helm3/strimzi-kafka-operator/values.yaml
2. `helm install strimzi strimzi/strimzi-kafka-operator -f strimzi.values.yaml` 

**Expected behavior**
It's expected that this version of the helm chart works with the latest version of the helm chart. However it appears that using the 0.31.1 image breaks the operator. It works when changing this to 0.31.0. 

**Environment (please complete the following information):**
 - Strimzi version: 0.31.0.
 - Installation method: Helm chart
 - Kubernetes cluster: Kubernetes 1.22
 - Infrastructure:  AWS EC2 Using kops

**YAML files and logs**
```
	HOSTNAME: strimzi-cluster-operator-69cf8c44bd-q6jqp
	STRIMZI_KAFKA_CONNECT_IMAGES: 3.1.0=quay.io/strimzi/kafka:0.31.1-kafka-3.1.0
3.1.1=quay.io/strimzi/kafka:0.31.1-kafka-3.1.1
3.2.0=quay.io/strimzi/kafka:0.31.1-kafka-3.2.0
3.2.1=quay.io/strimzi/kafka:0.31.1-kafka-3.2.1

	STRIMZI_KAFKA_IMAGES: 3.1.0=quay.io/strimzi/kafka:0.31.1-kafka-3.1.0
3.1.1=quay.io/strimzi/kafka:0.31.1-kafka-3.1.1
3.2.0=quay.io/strimzi/kafka:0.31.1-kafka-3.2.0
3.2.1=quay.io/strimzi/kafka:0.31.1-kafka-3.2.1

	KUBERNETES_PORT_443_TCP_PORT: 443
	STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE: quay.io/strimzi/kafka:0.31.1-kafka-3.2.1
	STRIMZI_OPERATOR_NAME: strimzi-cluster-operator-69cf8c44bd-q6jqp
	HOME: /home/strimzi
	MALLOC_ARENA_MAX: 2
	STRIMZI_OPERATOR_NAMESPACE: default

Exception in thread "main" io.strimzi.operator.common.InvalidConfigurationException: Failed to parse default container image configuration for Kafka from environment variable STRIMZI_KAFKA_IMAGES
	at io.strimzi.operator.cluster.ClusterOperatorConfig.parseKafkaVersions(ClusterOperatorConfig.java:396)
	at io.strimzi.operator.cluster.ClusterOperatorConfig.fromMap(ClusterOperatorConfig.java:210)
	at io.strimzi.operator.cluster.Main.main(Main.java:76)
Caused by: io.strimzi.operator.cluster.model.NoImageException: No image for version 3.1.2 in {3.1.0=quay.io/strimzi/kafka:0.31.1-kafka-3.1.0, 3.1.1=quay.io/strimzi/kafka:0.31.1-kafka-3.1.1, 3.2.0=quay.io/strimzi/kafka:0.31.1-kafka-3.2.0, 3.2.1=quay.io/strimzi/kafka:0.31.1-kafka-3.2.1}
	at io.strimzi.operator.cluster.model.KafkaVersion$Lookup.image(KafkaVersion.java:201)
	at io.strimzi.operator.cluster.model.KafkaVersion$Lookup.validateImages(KafkaVersion.java:237)
	at io.strimzi.operator.cluster.model.KafkaVersion$Lookup.validateKafkaImages(KafkaVersion.java:254)
	at io.strimzi.operator.cluster.ClusterOperatorConfig.parseKafkaVersions(ClusterOperatorConfig.java:382)
	... 2 more
```

--- a/charts/index.yaml
+++ b/charts/index.yaml
@@ -2,6 +2,34 @@ apiVersion: v1
 entries:
   strimzi-kafka-operator:
   - apiVersion: v2
+    appVersion: 0.31.1
+    created: "2022-09-19T15:24:42.699036+02:00"
+    description: 'Strimzi: Apache Kafka running on Kubernetes'
+    digest: 9b4843d0956ba3b6873c2dc9d06e57e31a2b166159db37fc40290657cc51e314
+    home: https://strimzi.io/
+    icon: https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/main/documentation/logo/strimzi_logo.png
+    keywords:
+    - kafka
+    - queue
+    - stream
+    - event
+    - messaging
+    - datastore
+    - topic
+    maintainers:
+    - name: Frawless
+    - name: ppatierno
+    - name: samuel-hawker
+    - name: scholzj
+    - name: tombentley
+    - name: sknot-rh
+    name: strimzi-kafka-operator
+    sources:
+    - https://github.com/strimzi/strimzi-kafka-operator
+    urls:
+    - https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.31.1/strimzi-kafka-operator-helm-3-chart-0.31.1.tgz
+    version: 0.31.1
+  - apiVersion: v2
     appVersion: 0.31.0
     created: "2022-09-02T15:20:11.31378+02:00"
     description: 'Strimzi: Apache Kafka running on Kubernetes'
@@ -1077,4 +1105,4 @@ entries:
     urls:
     - https://github.com/strimzi/strimzi-kafka-operator/releases/download/0.6.0/strimzi-kafka-operator-0.6.0.tgz
     version: 0.6.0
-generated: "2022-09-02T15:20:11.3101+02:00"
+generated: "2022-09-19T15:24:42.695444+02:00"

